devtools/server/actors/replay/connection.js

```diff
@@ -533,6 +537,11 @@ class Recording extends EventEmitter {
     }
   }
 
+  _onRecordingCreated(data) {
+    // Once this arrives, _then_ send API requeests to upload sourcemaps
```

Comment:
There's a separate PR open that removes this and actually does something with it.